lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimatable.kt

```diff
@@ -278,9 +284,12 @@ private class LottieAnimatableImpl : LottieAnimatable {
         val minProgress = clipSpec?.getMinProgress(composition) ?: 0f
         val maxProgress = clipSpec?.getMaxProgress(composition) ?: 1f
 
-        val dProgress = dNanos / 1_000_000 / composition.duration * speed
+        val speedValue: Float by derivedStateOf {
```

Comment:
What is the derivedStateOf doing here? This isn't a composable function.

Reply:
See my comment above -- https://github.com/airbnb/lottie-android/pull/2128#discussion_r975522959

I took a closer look at why making `speedValue` a val on L287 doesn't work as intended. The reason is that the rest of `onFrame` needs an updated `speedValue` each time `iteration` changes. So we would need a `var` on L287 and a _reassigment_ on L306. 

Based on this I decided that might further complicate `onFrame` which is already fairly complex so I reverted back to using a property for `speedValue`. If you would prefer the var + reassignment instead let me know.